src/views/tutorial-view/index.tsx

```diff
@@ -201,6 +202,17 @@ function TutorialView({
 		collectionId: collectionCtx.current.id,
 	})
 
+	/**
+	 * Display toast when progress changes to complete.
+	 */
+	useProgressToast({
```

Comment:
`useProgressToast` does the work, but we do of course need to hook it up in the view.